app/apps/cases/signals.py

```diff
@@ -0,0 +1,19 @@
+from apps.camunda.services import CamundaService
```

Comment:
Dedicated signals.py (en tests) toegevoegd

Reply:
Weet niet of ik een dedicated signals file overzichtelijk vind. Signals hebben direct te maken met het model en de instances die aangemaakt worden.

Reply:
Dit wordt aangeraden in de [officiële documentatie](https://docs.djangoproject.com/en/3.1/topics/signals/):
_Where should this code live?

Strictly speaking, signal handling and registration code can live anywhere you like, although it’s recommended to avoid the application’s root module and its models module to minimize side-effects of importing code.

In practice, signal handlers are usually defined in a signals submodule of the application they relate to._

Reply:
Dit wordt aangeraden in de [officiële documentatie](https://docs.djangoproject.com/en/3.1/topics/signals/):

> Where should this code live?
> 
> Strictly speaking, signal handling and registration code can live anywhere you like, although it’s recommended to avoid the application’s root module and its models module to minimize side-effects of importing code.
> 
> In practice, signal handlers are usually defined in a signals submodule of the application they relate to.